src/classes/Commands.ts

```diff
@@ -1228,15 +1228,28 @@ export = class Commands {
     }
 
     private tradesCommand(steamID: SteamID): void {
+        const now = moment().unix();
         const aDayAgo = moment().subtract(24, 'hour');
         const startOfDay = moment().startOf('day');
 
         let tradesToday = 0;
         let trades24Hours = 0;
         let tradesTotal = 0;
 
-        const offerData = this.bot.manager.pollData.offerData;
+        const timeSincePoll = this.bot.manager.pollData.timestamps;
+        const timeSince = timeSincePoll[Object.keys(timeSincePoll)[0]];
+
+        let reply = '';
+        reply += !timeSince ? '\n Just started. Waiting for the first trade.' : '\n Trades recorded from ';
 
+        const totalDaysInSeconds = now.valueOf() - timeSince;
+        let totalDays = Math.round((totalDaysInSeconds / 86400) * 1);
+        if (totalDays < 1) {
+            totalDays = 0;
+        }
+        reply += totalDays + ' ' + pluralize('day', totalDays) + ' ago';
```

Comment:
Pluralize can add the days infront of it by giving it true ad the third argument.

Reply:
so, `pluralize('day', totalDays, true)` ?